lib/rubocop/ast/processed_source.rb

```diff
@@ -159,6 +160,37 @@ def line_indentation(line_number)
           .length
       end
 
+      def node_tokens(node)
+        @node_tokens ||= {}.compare_by_identity
+        return @node_tokens[node] if @node_tokens[node]
+
+        begin_index = node_first_token_index(node)
+        end_index   = node_last_token_index(node)
+        @node_tokens[node] = tokens[begin_index..end_index]
+      end
+
+      def node_first_token(node)
+        node_tokens(node).first
+      end
+
+      def node_last_token(node)
+        node_tokens(node).last
```

Comment:
Are we sure we need the method at all? Implementation is not efficient...

Reply:
Same as above